src/common/snippets/include/snippets/op/brgemm.hpp

```diff
@@ -55,7 +55,7 @@ class Brgemm : virtual public modifier::MemoryAccess, public ov::op::Op {
 protected:
     ov::element::Type get_output_type() const;
     std::vector<ov::PartialShape> get_planar_input_shapes(const std::vector<ov::Input<ov::Node>>& inputs) const;
-    ov::PartialShape get_output_partial_shape(const std::vector<ov::PartialShape>& input_shapes) const;
+    ov::PartialShape infer_output_partial_shape(const std::vector<ov::PartialShape>& input_shapes) const;
```

Comment:
Note: `get_output_partial_shape()` - is a `Node` s non-virtual method that was hidden by this declaration, so I had to re-name it to `infer_output_partial_shape`